packages/app/src/sandbox/eval/transpilers/vue/loader.js

```diff
@@ -122,11 +122,12 @@ export default function(content: string, loaderContext: LoaderContext) {
     ts: ['ts-loader'],
     typescript: ['ts-loader'],
     pug: ['pug-loader'],
+    coffee: ['coffee-loader'],
   };
 
   const loaders = Object.assign({}, defaultLoaders, codeSandboxLoaders);
   const preLoaders = {};
-  const postLoaders = {};
+  const postLoaders = { coffee: 'babel-loader' };
```

Comment:
This seemed like the clencher... only by having this could I get the `<script lang='coffee'>` sections of the `.vue` files to work.